src/compile/legend/parse.ts

```diff
@@ -171,7 +168,7 @@ function getProperty<K extends keyof VgLegend>(
           legend,
           legendConfig,
           channel,
-          scaleType: model.getScaleComponent(channel).get('type')
+          scaleType: scaleType
```

Comment:
remove `: scaleType`

Reply:
done already, I'll make this change in your PR